src/pages/policy/output/InequalityImpact.jsx

```diff
@@ -51,7 +51,7 @@ export default function InequalityImpact(props) {
         },
         yaxis: {
           title: "Relative change",
-          tickformat: ",.0%",
+          tickformat: ",.2%",
```

Comment:
We might later want to do two decimal points, but we'd do it in both the axis and the hovercard.
```suggestion
          tickformat: ",.1%",
```